desktopIconsIntegration.js

```diff
@@ -55,16 +55,17 @@
  *
  *******************************************************************************/
 
-import {GLib} from './dependencies/gi.js';
-import {Main} from './dependencies/shell/ui.js';
-import {ExtensionUtils} from './dependencies/shell/misc.js';
-
-import {DockManager} from './docking.js';
+import GLib from 'gi://GLib';
+import * as Main from 'resource:///org/gnome/shell/ui/main.js';
+import * as ExtensionUtils from 'resource:///org/gnome/shell/misc/extensionUtils.js';
+import {Extension} from 'resource:///org/gnome/shell/extensions/extension.js';
```

Comment:
I think you can still use our way to load deps, no?

Reply:
Yes. Although that would beat the point of being the same independent file for all extensions. :smile:

In any case, added a commit to import dependencies your way.